zwave_js_server/model/node/event_model.py

```diff
@@ -103,11 +105,7 @@ class NotificationEventModel(BaseNodeEventModel):
 
     event: Literal["notification"]
     ccId: CommandClass
-    args: Union[
-        NotificationNotificationArgsDataType,
-        EntryControlNotificationArgsDataType,
-        PowerLevelNotificationArgsDataType,
-    ]
+    args: NotificationNotificationArgsDataType | EntryControlNotificationArgsDataType | PowerLevelNotificationArgsDataType
```

Comment:
I think we can use parenthesis to avoid a too long line.

Reply:
thanks, weird that black didn't do that automatically